docs/operation-guides/50-kiali-grafana.md

```diff
@@ -1,61 +1,52 @@
 ---
 title: Access and Expose Kiali and Grafana
-type: Configuration
 ---
 
-By default, Kyma does not expose Kiali and Grafana. However, you can still access them using port forwarding. Also, read [Expose Kyma UIs securely](http://tbd) to learn how to expose Kiali and Grafana securely using an identity provider of your choice.
+By default, Kyma does not expose Kiali and Grafana. However, you can still access them using port forwarding. If you want to expose Kiali and Grafana securely, use an identity provider of your choice.
 
-### Prerequisites
+## Prerequisites
 
 - You have defined the kubeconfig file for your cluster as default (see [Kubernetes: Organizing Cluster Access Using kubeconfig Files](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)).
+- To expose the services securely with OAuth, you have a registered OAuth application with one of the [supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider#github-auth-provider).
+
+## Access Kiali and Grafana
+
+### Steps
+
+1. To forward a local port to a port on the service's Pod, run the following command:
 
 <div tabs>
   <details>
   <summary>
   Kiali
   </summary>
 
-  ### Steps
-    
-  To access Kiali, do the following:
-
-  1. Run the following command to forward a local port to a port on the Kiali Pod:
   ```bash
   kubectl -n kyma-system port-forward svc/kiali-server 20001:20001
   ```
-  >**NOTE:** kubectl port-forward does not return. You will have to cancel it with Ctrl+C if you want to stop port forwarding.
-
-  2. Open http://localhost:20001 in your browser. You see the Kiali UI.
 
   </details>
   <details>
   <summary>
   Grafana
   </summary>
 
-  ### Steps
-    
-  To access Grafana, do the following:
-
-  1. Run the following command to forward a local port to a port on the Grafana Pod:
   ```bash
   kubectl -n kyma-system port-forward svc/monitoring-grafana 3000:80
   ```
-  >**NOTE:** kubectl port-forward does not return. You will have to cancel it with Ctrl+C if you want to stop port forwarding.
-
-  2. Open http://localhost:3000 in your browser. You see the Grafana UI.
 
   </details>
 
 </div>
 
-## Expose Kiali and Grafana Securely
 
-To make Kiali and Grafana permanently accessible, expose the services securely using [oauth2_proxy](https://oauth2-proxy.github.io/).
+>**NOTE:** kubectl port-forward does not return. You will have to cancel it with Ctrl+C if you want to stop port forwarding.
```

Comment:
Sth is missing here.. Did you mean: ```If `kubectl port-forward` does not return any results, cancel it with `Ctrl+C` to stop port forwarding.```?

Reply:
The dev explained to me that this process does not stop and return to the input prompt in the terminal, so to stop if from running indefinitely users must cancel it manually. 

I assumed "does not return" is normal dev lingo for that and would be understood, but I see now that we need to spell it out clearer :D

Reply:
Now that I check it seems to be a wording that is quite often used (to my surprise) -> https://cloud.google.com/community/tutorials/writing-prometheus-metrics-bigquery. Sorry for confusion ;)

Reply:
alright, then let's keep it